ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadManager.swift

```diff
@@ -229,6 +236,9 @@ public class ResourceDownloadManager {
       // Perform common 'can download' check.  We need positive reachability and no prior download queue.
       // The parameter facilitates error logging.
       if !downloader.canExecute(.simpleBatch(dlBatch)) {
+        let error = NSError(domain: "Keyman", code: 0,
+                          userInfo: [NSLocalizedDescriptionKey: "Download queue is either busy or lacks internet connection"])
```

Comment:
Is this a message for the end user? I have no idea what it means -- nor what action could be taken as a result! 😁

Reply:
Upon inspection... from `InstalledLanguagesViewController, starting at line 353:

```
  private func lexicalModelDownloadFailed() {
    log.info("lexicalModelDownloadFailed: InstalledLanguagesViewController")
    restoreNavigation()
    
    let title = "Dictionary Download Error"
    navigationController?.setToolbarHidden(true, animated: true)
    
    let alertController = UIAlertController(title: title, message: "",
                                            preferredStyle: UIAlertController.Style.alert)
    alertController.addAction(UIAlertAction(title: "OK",
                                            style: UIAlertAction.Style.cancel,
                                            handler: nil))
    
    self.present(alertController, animated: true, completion: nil)
  }
```

It doesn't use this error string; it's just for the logs.  Likewise for the other error case.

Reply:
Not that "Dictionary Download Error" is particularly informative either, mind you.